src/main/java/com/chapter3/StringUtil.java

```diff
@@ -0,0 +1,94 @@
+package com.chapter3;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Scanner;
+
+
+public class StringUtil {
+
+    private static String getEnteredText(String enteredString) {
+        return enteredString;
+    }
+
+    private static int getTextLength(String enteredString) {
+        return enteredString.length();
+    }
+
+    private static int countLetters(String enteredString) {
+        int count = 0;
+
+        for (int i = 0; i < enteredString.length(); i++) {
+            if (enteredString.charAt(i) == 'a') {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    private static void findLetterIndex(String enteredString) {
+        int start = enteredString.indexOf('a');
```

Comment:
or rename function to findALetterIndex, or path 'a' or any other letter as parameter